lib/client.js

```diff
@@ -54,7 +54,7 @@ export default class Client {
     this.baseUrl = baseUrl;
     return this;
   }
-  promiseProxy(f, req) {
+  promiseProxy(f, args) {
     if (this.promises || !f) {
```

Comment:
What is `f` supposed to be?

Reply:
It's a function, it's either a promise resolver, or a callback passed in at the call site.